merge_rgba/__init__.py

```diff
@@ -94,12 +94,17 @@ def merge_rgba_tool(sources, outtif, bounds=None, res=None, precision=7,
         for idx, dst_window in dstrast.block_windows():
 
             left, bottom, right, top = dstrast.window_bounds(dst_window)
-            blocksize = ((dst_window[0][1] - dst_window[0][0]) *
-                         (dst_window[1][1] - dst_window[1][0]))
+            if isinstance(dst_window, rasterio.windows.Window):
```

Comment:
We'll get an `AttributeError` for rasterio<1.0. How about test for being an instance of `tuple` first and then try the above, catching any error?